apps/server/src/infra/calendar/service/calendar.service.ts

```diff
@@ -43,6 +103,27 @@ export class CalendarService {
 			});
 	}
 
+	async deleteEventsByScopeId(scopeId: EntityId): Promise<void> {
+		const request = this.httpService.delete(`/scopes/${scopeId}`, {
+			headers: {
+				Authorization: scopeId,
+				Accept: 'Application/json',
+			},
+			timeout: this.timeoutMs,
+		});
+
+		const resp = await firstValueFrom(request).catch((error) => {
+			throw new InternalServerErrorException(
+				null,
+				ErrorUtils.createHttpExceptionOptions(error, 'CalendarService:findEvent')
+			);
+		});
+
+		if (resp.status !== 204) {
+			throw new Error(`invalid HTTP status code in a response from the server instead of 204`);
```

Comment:
I'd add information on what (unexpected) status code we've received:
```suggestion
			throw new Error(`invalid HTTP status code in a response from the server - ${resp.status} instead of 204`);
```